### ISSUE TITLE
What is exactly is Kubernetes, Is that a rare pokemon?

### ISSUE THREAD

**Issue by vipulgupta2048** [asetalias/Logistics-and-Event-Contributions#42]
# What is exactly is Kubernetes, Is that a rare pokemon?

**Type**
- [ ] Workshop / Hands on
- [ ] Guest Lecture
- [ ] Webinar
- [x] Talk

**Abstract**
Rare Pokemon are hard to find. That's the same goes for Kubernetes. Except it is everywhere in today's tech space. It's blowing up faster than the next JS framework. But, it's not a bubble. It's a revolutionary tool that is gonna change how we look at Cloud-native in its entirety. 

5 years ago, Kubernetes was released. It is an open-source container orchestration system for automating application deployment, scaling, and management. Originally designed by Google, now maintained by the Cloud Native Computing. It's great and you might have read this definition many times. But, you have this feeling you still don't have a grip about what it means. More importantly, why you should bother. Not worry anymore, because we will be talking about all these points and more to understand what K8's really is. What are the clusters? Why do we need an orchestration system for containers and much more?   

**Pre-requisites**

- Will Update

**Expected duration**

An hour, it will be brief and a ton of fun. 

**Level**

_Beginner/Intermediate/Advanced_

**Resources**

Will Update.

**Speaker Bio**

Vipul is a 1x OSS Software Engineer, 2x loves the community and party parrots. And, runs his own initiative for writing docs for startups and OSS organizations called Mixster.

**Social Links to your profile**  
https://vipulgupta2048.me and @vipulgupta2048 

**Month**
January, like the college, reopens. 

@Sc0rpi0n101 @sjha2048 @TanyaaCJain @thisisayush Let me know if we can have lightening like a session on this. To get people introduced to this. I know its way above the league for some students but no harm in flirting with new stacks.

**Comment by sjha2048** [comment]
I guess last week of Thursday is viable, or we can merge two events.

**Comment by thisisayush** [comment]
@vipulgupta2048 I would love a lightening session on this, let's plan it

**Comment by thisisayush** [comment]
Also +1 for the talk

**Comment by thisisayush** [comment]
@vipulgupta2048 Let's do this on a webinar or something, you available anytime soon?

**Comment by vipulgupta2048** [comment]
Yeah, go for it. This might be the push I need to get this done as well.

**Comment by aniketmaithani** [comment]
+1 for this. Can you share any links/documentation so that beginners can get a sneak peak.

**Comment by vipulgupta2048** [comment]
I ain't an expert myself, but I had to write docs for something at a very close intersection with K8's. Hence, I thought why not write about it. So if you like The Office as much as I do. You can check this all in one starter guide that I made, https://mixstersite.wordpress.com/2020/01/07/demystifying-kubernetes-k8s-the-office-version/

**Comment by Sc0rpi0n101** [comment]
Great
When would you like to schedule it?

**Comment by vipulgupta2048** [comment]
This Saturday/Sunday?

**Comment by Sc0rpi0n101** [comment]
This Saturday is the panel discussion around 12 PM.
So, we can do this during the night. or we can do it anytime you're free sunday.

**Comment by vipulgupta2048** [comment]
Sunday it is, please do make a calendar event.

**Comment by Sc0rpi0n101** [comment]
Yeah, sorry. Would you be available anytime this week?
